### PR TITLE
[FRONT-317] Fix DU member count badge not shown in marketplace

### PR DESCRIPTION
**Pull request by juhah**
Fixes DU stats loading on product page.

### CODE REVIEW

File: app/src/marketplace/modules/productList/actions.js [hunk 1]
```diff
@@ -64,22 +64,38 @@ const doGetProducts = (replace: ?boolean = false, dispatch: Function, getState:
...
 // https://gist.github.com/krstffr/245fe83885b597aabaf06348220c2fe9
-const doGetProductsDebounced = debounce(doGetProducts, 500)
+const doGetProductsDebounced = debounce((
+    { replace, onSuccess, onError }: GetProducts,
+    dispatch: Function,
+    getState: () => StoreState,
+) => (
+    doGetProducts(replace, dispatch, getState)
+        .then(onSuccess)
+        .then(onError)
```
Comment (mondoreale) [inline app/src/marketplace/modules/productList/actions.js:91]: Won't it hit `onError` despite being resolved successfully?

File: app/src/marketplace/containers/Products/index.jsx [hunk 2]
```diff
@@ -41,41 +42,62 @@ const Products = () => {
...
 
     const loadProducts = useCallback(() => dispatch(getProducts()), [dispatch])
 
     const onFilterChange = useCallback((filter: Filter) => {
         dispatch(updateFilter(filter))
         dispatch(getProducts(true))
-    }, [dispatch])
+            .then((productIds) => {
+                if (isMounted()) {
+                    loadDataUnionStats(productIds)
```
Comment (mondoreale) [inline app/src/marketplace/containers/Products/index.jsx:61]: So, the solution is to reload DU stats after filtering/searching. Cool.

**Review by mondoreale — APPROVED** [review]

**Comment by fonty1** [comment]
Did Chromatic pick up the bug?

**Review by fonty1 — APPROVED** [review]

File: app/src/marketplace/modules/productList/actions.js (hunk 1 above)
Comment (juhah) [reply app/src/marketplace/modules/productList/actions.js]: yes 🙈 was meant to be `catch`

**Comment by juhah** [comment]
> Did Chromatic pick up the bug?

@fonty1 No, it wouldn't show up there because the problem was that the stats fetching didn't work properly on the Marketplace listing. Elsewhere it was ok (probably my oversight from earlier refactoring).

**Pull request merged**